packages/p2p/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "deriv-p2p",
+  "version": "1.0.5",
+  "description": "p2p cashier",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"No test specified\"",
+    "build": "rimraf lib && webpack --mode=production --optimize-minimize --progress",
```

Comment:
Please remove `--mode` arguments from scripts. These should be handled inside the webpack config.

Reply:
sure